api/src/controllers/flows.ts

```diff
@@ -13,18 +13,19 @@ const router = express.Router();
 
 router.use(useCollection('directus_flows'));
 
-const webhookFlowHandler = asyncHandler(async (req, res) => {
+const webhookFlowHandler = asyncHandler(async (req, res, next) => {
 	const flowManager = getFlowManager();
 
 	const result = await flowManager.runWebhookFlow(`${req.method}-${req.params.pk}`, req.body, {
 		accountability: req.accountability,
 	});
 
-	res.json(result);
+	res.locals.payload = result;
+	return next();
 });
 
-router.get(`/trigger/:pk(${UUID_REGEX})`, webhookFlowHandler);
-router.post(`/trigger/:pk(${UUID_REGEX})`, webhookFlowHandler);
+router.get(`/trigger/:pk(${UUID_REGEX})`, webhookFlowHandler, respond);
```

Comment:
This introduces caching to flows triggered by `GET` requests. As flows might have side effects like _sending a notification_, we should at least make sure to document this properly.

Reply:
Yup! They can still send `Cache-Control: no-store` to get around the cache though, so I think that's OK